docs/sources/flow/concepts/config-language/_index.md

```diff
@@ -42,6 +42,12 @@ BLOCK_NAME {
 }
 ```
 
+[River is designed][RFC] with the following requirements in mind:
```

Comment:
The additions in this file are a result of folding the previous `concepts/configuration_language.md` into this file, so we can avoid two pages talking about the same.

Reply:
Given the folding of that file into this one, should we add a redirect for that page to this?

Reply:
Gotcha - I've added all the aliases that the deleted file has.

Reply:
Nice, I think the only one that is missing is the redirect for the deleted page itself. A relative alias that redirects `/docs/agent/flow/concepts/configuration_language/` to `/docs/agent/<AGENT_VERSION>/flow/concepts/config-language/`

Reply:
Gotcha, added it and verified that `curl http://localhost:3003/docs/agent/latest/flow/concepts/configuration_language/` works. Hope that's what you meant 😅